### PR TITLE
implement

### PR DESCRIPTION
**Pull request by divszz**
## Description
Implemented GitHub action to tcet-linux-qogir-theme repository

## Screenshots (if applicable)
N/A

## Checklist
- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/tcet-opensource/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] I have read and followed the **Contributing Guidelines**

### CODE REVIEW

**Review by 0xAtharv — CHANGES_REQUESTED** [review]
File structure should be `.github/workflows/release.yml`

File: release.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          
+      - name: Release Tar File with latest
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "*.tar.gz,checksum"
+          generateReleaseNotes: false
+          name: "TCET-Linux Wallpaper"
+          tag: latest
+
+       - name: Release Tar File
```
Comment (0xAtharv) [inline release.yml]: indextation is wrong . Extra space in line 42

**Review by 0xAtharv — CHANGES_REQUESTED** [review]

**Review by harshau007 — APPROVED** [review]
LGTM!!!!

**Review by Akash6222 — APPROVED** [review]
GTG

**Review by Akash6222 — APPROVED** [review]
solved

**Pull request merged**